src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,41 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_SIZE = 10;
+    private K[] keyArray = (K[]) new Object[MAX_STORAGE_SIZE];
+    private V[] valueArray = (V[]) new Object[MAX_STORAGE_SIZE];
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < keyArray.length; i++) {
+            if ((keyArray[i] == null && valueArray[i] == null)
+                    || (key == null && keyArray[i] == null)
+                    || (key != null && key.equals(keyArray[i]))) {
+                keyArray[i] = key;
+                valueArray[i] = value;
+                return;
+            }
+        }
     }
 
     @Override
     public V get(K key) {
+        for (int i = 0; i < keyArray.length; i++) {
+            if ((key == null && keyArray[i] == null) || (key != null && key.equals(keyArray[i]))) {
+                return valueArray[i];
+            }
+        }
         return null;
     }
 
     @Override
     public int size() {
-        return -1;
+        int count = 0;
+        for (V v : valueArray) {
+            if (v != null) {
+                count++;
+            }
+        }
+        return count;
```

Comment:
Add size variable for your class.
```suggestion
        return size;
```